### DISCUSSION TITLE
I want to join your team

### DISCUSSION THREAD

**Discussion by Sachin9822** [ROM-builders/temporary#14306] [Q&A]
Github link:- https://github.com/Sachin9822/temporary
telegram Username:- Sachin9822
I'm building for evolution X Tissot and Project Blaze
successful Builds:- 
https://cirrus-ci.com/task/5199459356246016
https://cirrus-ci.com/build/5397831883161600

https://cirrus-ci.com/task/5680522671161344
https://cirrus-ci.com/task/5584778522001408

Local Manifest:- https://github.com/Sachin9822/local_manifest/blob/evo/local_manifest.xml
DT :- https://github.com/Sachin9822/android_device_xiaomi_tissot
DT common:- https://github.com/Sachin9822/device_xiaomi_msm8953-common
vendor tree:- https://github.com/Sachin9822/vendor_xiaomi_tissot
vendor common tree:- https://github.com/Sachin9822/vendor_xiaomi_msm8953-common

**Comment by Sachin9822** [comment]
> Github link:- https://github.com/Sachin9822/temporary telegram Username:- Sachin9822 I'm building for evolution X Tissot successful Builds:- https://cirrus-ci.com/task/5199459356246016 https://cirrus-ci.com/build/5397831883161600
> 
> Local Manifest:- https://github.com/Sachin9822/local_manifest/blob/evo/local_manifest.xml DT :- https://github.com/Sachin9822/android_device_xiaomi_tissot DT common:- https://github.com/Sachin9822/device_xiaomi_msm8953-common vendor tree:- https://github.com/Sachin9822/vendor_xiaomi_tissot vendor common tree:- https://github.com/Sachin9822/vendor_xiaomi_msm8953-common

@Apon77

**Reply by Apon77** [reply]
Please check #learn and #team instruction in telegram

**Reply by Sachin9822** [reply]
@Apon77 sir learned to cherry-pick and added pixel launcher in ProjectBlaze 
and here is my [script](https://github.com/Sachin9822/temporary/blob/main/build_rom.sh)

**Reply by Apon77** [reply]
need more time because there is no thing which says you learned those, please wait

**Reply by Sachin9822** [reply]
Ok No problem i will try to improve

**Reply by Sachin9822** [reply]
@Apon77 
https://github.com/Sachin9822/android_device_xiaomi_tissot/commits/ProjectBlaze

Can i join the team now ?

**Reply by Sachin9822** [reply]
@Apon77 i have also built PE A13 Plus 
https://cirrus-ci.com/build/6471195684503552

**Reply by Apon77** [reply]
Please check telegram message

**Comment by Sachin9822** [comment]
Done
